### ISSUE TITLE
Missing sanitizer support libraries

### ISSUE THREAD

**Issue by gpakosz** [kerberizer/llvm-svn#5]
I installed clang-svn-3.8.0svn_r247119-1 but it lacks sanitizers support libraries:

With the following empty `main.c` program:

```
int main()
{
  return 0;
}
```

I'm getting:

```
$ clang -fsanitize=thread main.c
/usr/bin/ld: cannot find /usr/bin/../lib/clang/3.8.0/lib/linux/libclang_rt.tsan-x86_64.a: No such file or directory

$ clang -fsanitize=address main.c
/usr/bin/ld: cannot find /usr/bin/../lib/clang/3.8.0/lib/linux/libclang_rt.asan-x86_64.a: No such file or directory
```

etc...

**Comment by kerberizer** [comment]
@gpakosz, thank you very much for your report! Could you please update your packages from the repo and additionally install the new `clang-compiler-rt-svn` package. This should hopefully solve the issue. You can also use the newly created `llvm-toolchain-svn` group to conveniently install/update the whole set of packages except for `llvm-ocaml-svn`, e.g. `pacman -Sy llvm-toolchain-svn`.

**Comment by kerberizer** [comment]
Assumed fixed by a5421a204595847b8c28cb7e801ea21d2a0113cd. Closing.
